data/net/minecraft/world/level/levelgen/feature/StructureFeature.mapping

```diff
@@ -1,11 +1,58 @@
 CLASS net/minecraft/world/level/levelgen/feature/StructureFeature
+	METHOD <init> (Lcom/mojang/serialization/Codec;)V
+		ARG 1 codec
+	METHOD createStart (Lnet/minecraft/world/level/ChunkPos;IJ)Lnet/minecraft/world/level/levelgen/structure/StructureStart;
+		ARG 1 pos
+		ARG 2 references
```

Comment:
ChunkGenerator#createStructure
It says `int integer13 = structureStart12 != null ? structureStart12.getReferences() : 0;`, so... `getReferences` is `references` because the field which will be got is named `references`.